spec/unit/mutant/env/boostrap_spec.rb

```diff
@@ -155,14 +152,11 @@ def name
         end
       end
 
-      it 'warns via reporter' do
-        expected_warnings = [
+      before do
+        expected_warning =
           "Class#name from: #{invalid_class.inspect} returned Object. #{Mutant::Env::SEMANTICS_MESSAGE}"
-        ]
 
-        expect { subject }
-          .to change { config.reporter.warn_calls }
-          .from([]).to(expected_warnings)
+        expect(config.reporter).to receive(:warn).with(expected_warning).and_return(config.reporter)
```

Comment:
Given there are 3 placed in this spec that do the same thing, what about creating a single method that creates the expectation given `expected_warnings` and use it here and elsewhere?